hcp/preprocessing.py

```diff
@@ -214,7 +214,7 @@ def interpolate_missing(inst, subject, data_type, hcp_path,
     is_raw = isinstance(inst, (mne.io.Raw,
                                mne.io.RawArray,
                                mne.io.bti.bti.RawBTi))
-    is_epochs = isinstance(inst, (mne.Epochs, mne.EpochsArray))
+    is_epochs = isinstance(inst, mne.BaseEpochs)
```

Comment:
actually, to make sure that you are compliant with older versions of MNE, you should do something like this: https://github.com/autoreject/autoreject/blob/master/autoreject/utils.py#L61-L64